lib/brain.js

```diff
@@ -743,35 +743,11 @@ Brain.prototype.timeoutToScannerCancel = function timeoutToScannerCancel (stateM
   }, SCANNER_TIMEOUT)
 }
 
-const readFailedFrames = callback => failedScansTmpdirs =>
-  Promise.all(_.map(
-    dir => fsPromises.readdir(dir)
-      .catch(err => {
-        console.error(err)
-        return []
-      })
-      .then(_.map(fname => path.join(dir, fname))),
-    failedScansTmpdirs
-  ))
-  .then(_.flow(
-    _.flatten,
-    failedScans => _.chunk(_.round(_.size(failedScans)/6), failedScans),
-    _.map(failedScanGroup =>
-      fsPromises.readFile(_.last(failedScanGroup))
-        .then(callback)
-        .catch(console.error)
-    ),
-    ps => Promise.all(ps),
-  ))
-  .catch(console.error)
-
 Brain.prototype.scanPDF = function scanPDF () {
   const customer = this.customer
   // TODO: Change the LED code to support genmega
   this.scanBayLightOn()
-  const saveFailedScans = this.shouldSaveQrAttempts ?
-    readFailedFrames(photo => idCardData.addIdDataPhoto(photo)) :
-    null
+  const idCardStillsCallback = photo => idCardData.addIdDataPhoto(photo)
```

Comment:
This will use the first 5 failed frames (which could be the first 5s of a 30s-60s interval), which is not necessarily a good sample.